src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -339,6 +342,38 @@ const MissingItemFormEdit = () => {
                   </Grid>
                 </Grid>
               )}
+              {/* Display a chip for items with statuses other than "active" */}
+              {formData.status.toLowerCase() !== 'active' &&
+                formData.status.toLowerCase() !== 'found' && (
+                  <Grid container xs={9.7} className={styles.foundContainer} rowGap={2}>
+                    <Grid container item xs={12} md={6} rowGap={2}>
+                      <Grid item xs={12}>
+                        <Chip
+                          className={styles.largeChip}
+                          sx={{
+                            height: 'auto',
```

Comment:
Can we move height: 'auto' into the CSS largeChip class?  I know the mui override section here needs to be in the sx, but this line doesn't as far as I can tell.

Reply:
Sure